contracts/rules/TransferRules.sol

```diff
@@ -56,17 +57,26 @@ contract TransferRules is ITransferRules, ManualApproval, Whitelisted {
     * @param value The amount of tokens to send.
     */
     function doTransfer(address from, address to, uint256 value) external onlySRC20 returns (bool) {
+        require(
```

Comment:
Just to notice - this is `authorize` condition here, maybe call it if we keep them?